schema/manifest_backwards_compatibility_test.go

```diff
@@ -110,14 +110,14 @@ func TestBackwardsCompatibilityManifestList(t *testing.T) {
 			fail: false,
 		},
 	} {
-		sum := sha256.Sum256([]byte(tt.manifest))
+		sum := sha256.Sum256([]byte(tt.manifestlist))
```

Comment:
The digest open is manifeslist, so I think there is nothing wrong with the use of manifestlist here.